llvm/include/llvm/ADT/SmallPtrSet.h

```diff
@@ -231,14 +231,17 @@ class SmallPtrSetImplBase : public DebugEpochBase {
 protected:
   /// swap - Swaps the elements of two sets.
   /// Note: This method assumes that both sets have the same small size.
-  void swap(SmallPtrSetImplBase &RHS);
+  void swap(const void **SmallStorage, const void **RHSSmallStorage,
+            SmallPtrSetImplBase &RHS);
 
-  void CopyFrom(const SmallPtrSetImplBase &RHS);
-  void MoveFrom(unsigned SmallSize, SmallPtrSetImplBase &&RHS);
+  void CopyFrom(const void **SmallStorage, const SmallPtrSetImplBase &RHS);
```

Comment:
Optional: as you are changing the signature, you can rename MoveFrom to moveFrom to improve the naming consistency.